src/main/java/core/basesyntax/Randomable.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public interface Randomable {
+    ColorSupplier colorSupplier = new ColorSupplier();
+
+    Random random = new Random();
+
+    void generateRandomParameters();
+
```

Comment:
You don`t need this interface

Reply:
I dont really understand how can i generate random parameters without this interface. Maybe you can give a hint ?